WordPress/src/main/java/org/wordpress/android/ui/DualPaneFragment.java

```diff
@@ -0,0 +1,31 @@
+package org.wordpress.android.ui;
+
+import android.os.Build;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.View;
+
+import org.wordpress.android.R;
+
+/**
+ * Superclass used for shared logic of fragments with dual pane layout support.
+ */
+public class DualPaneFragment extends Fragment {
```

Comment:
So for a fragment to work within a dual pane layout it will have to descend from `DualPaneFragment`?